zip-0226.rst

```diff
@@ -25,40 +25,26 @@ The key word "MUST" in this document is to be interpreted as described in RFC 21
 
 The term "network upgrade" in this document is to be interpreted as described in ZIP 200 [#zip-0200]_.
 
-The terms "Orchard" and "Action" in this document are to be interpreted as described in
-ZIP 224 [#zip-0224]_.
+The terms "Orchard" and "Action" in this document are to be interpreted as described in ZIP 224 [#zip-0224]_.
 
-We define the following additional terms:
-
-- Asset: A type of note that can be transferred on the Zcash block chain, identified by the :math:`\mathsf{AssetId}` parameter.
+The terms "Asset", "Custom Asset" and "Wrapped Asset" in this document are to be interpreted as described in ZIP 227 [#zip-0227]_.
 
-  - ZEC is the default (and currently the only defined) Asset for the Zcash mainnet.
-  - TAZ is the default (and currently the only defined) Asset for the Zcash testnet.
-  - We use the term "Custom Asset" to refer to any Asset other than ZEC and TAZ.
+We define the following additional terms:
 
-- Native Asset: a Custom Asset with issuance defined on the Zcash block chain.
-- Wrapped Asset: a Custom Asset with native issuance defined outside the Zcash block chain.
-- Split Input: an Action input used to ensure that the output note of that Action is of a validly issued :math:`\mathsf{AssetBase}` when there is no corresponding real input note, in situations where the number of outputs are larger than the number of inputs. See formal definition in `Split Notes`_.
+- Split Input: an Action input used to ensure that the output note of that Action is of a validly issued :math:`\mathsf{AssetBase}` (see [#zip-0227-assetidentifier]_) when there is no corresponding real input note, in situations where the number of outputs are larger than the number of inputs. See formal definition in `Split Notes`_.
 - Split Action: an Action that contains a Split Input.
 
 Abstract
 ========
 
-ZIP 226 and ZIP 227 propose in conjunction the Zcash Shielded Assets (ZSA) protocol — a set
-of protocol features that enable the creation, transfer, and burn of Custom Assets on the Zcash chain.
-
-Creation of such Assets is defined in ZIP 227 [#zip-0227]_. Transfer and burn of such Assets is defined
-in ZIP 226 [#zip-0226]_. The ZSA protocol is proposed to be instantiated by a modification to the
-Orchard protocol, as specified in these ZIPs (although it has been designed with adaption
-to possible future shielded protocols in mind).
+The ZIP (ZIP 226) proposes the Zcash Shielded Assets (ZSA) protocol, in conjunction with ZIP 227 [#zip-0227]_. The ZSA protocol is an extension of the Orchard protocol that enables the creation, transfer and burn of custom Assets on the Zcash chain. The creation of such Assets is defined in ZIP 227 [#zip-0227]_, while the transfer and burn of such Assets is defined in this ZIP (ZIP 226).
+While the proposed ZSA protocol is a modification to the Orchard protocol, it has been designed with adaption to possible future shielded protocols in mind.
```

Comment:
```suggestion
While the proposed ZSA protocol is a modification to the Orchard protocol, it has been designed with adaptation to possible future shielded protocols in mind.
```

See [this link](https://grammarist.com/usage/adaption-adaptation/), for example. 

> Adaption is when something has been changed by its environment. Adaptation is changing or updating something to make it better or more functional in a different setting.

Using adaptation might be better?

Reply:
Agreed @vivek-arte.
`adaption` was part of the original doc so I stuck to it, but I was also tempted to replace. Let's definitely switch :)